supplier_api_tester/supplier_api_tester/utils/adapters.py

```diff
@@ -8,6 +11,29 @@
 from ..models import ApiError, Booking, DailyAvailability, DailyVariants, Product, Reservation, Timeslot
 
 
+def check_base64(item):
+    try:
+        return base64.b64encode(base64.b64decode(item)).decode() == item
+    except binascii.Error:
+        return False
+
+def booking_pdf_helper(booking: Booking, raw_response:Request):
+    if booking.barcode_position == 'order':
+        if not check_base64(booking.barcode):
+            raise FailedTest(
+                message = "Invalid barcode field encoding",
```

Comment:
Maybe something like `Error while decoding (base64) PDF voucher for the order` would be more descriptive to developers on where the issue is.